src/main/resources/application.yml

```diff
@@ -41,14 +49,13 @@ spring:
     show-sql: false
     defer-datasource-initialization: true
     hibernate:
-      ddl-auto: update
+      ddl-auto: none
       default_schema: casesvc
     properties:
       hibernate:
-        hbm2ddl:
-          schema_filter_provider: uk.gov.ons.ctp.response.casesvc.CustomProvider
-        id:
```

Comment:
This Custom provider originally thought to do something is not needed as far as I know (it has been removed from sample and collection exercise)

Reply:
That's correct, it was removed from sample in favour of Liquibase handling migrations. The only reason this had been introduced was an attempt to stop Hibernate from creating certain objects. That did not work as it does not honour the `dd-auto: update` strategy.